react/src/Containers/RestaurantsIndexContainer.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import RestaurantIndexTile from '../Components/RestaurantIndexTile'
+
+class RestaurantsIndexContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurants: []
+    }
+  }
+
+  componentDidMount() {
+    fetch(`/api/v1/restaurants`)
+    .then(response => {
+      return response.json()
+    })
+    .then(response => response.restaurants)
+    .then(restaurants => {
+      this.setState({restaurants})
+    })
+  }
+  render() {
+
+    let restaurants = this.state.restaurants.map(r => {
+      return(
+        <RestaurantIndexTile
+          key={r.id}
+          name={r.name}
+        />
+      )
+    })
+
+    return(
+      <div>
+        {restaurants}
+      </div>
+    )
+  }
+}
+
```

Comment:
Again, nice work creating this container. It does a great job of adhering to our design patterns!